exercises/practice/phone-number/.meta/lib/example.dart

```diff
@@ -13,27 +13,40 @@ class PhoneNumber {
       phoneNumber = phoneNumber.replaceAll(match.group(0)!, "");
     });
 
-    if (phoneNumber.contains(RegExp(r'[a-zA-Z]'))) throw FormatException('letters not permitted');
+    if (phoneNumber.contains(RegExp(r'[a-zA-Z]')))
+      throw FormatException('letters not permitted');
```

Comment:
I don't think a new line is needed here.

Reply:
That appears to have been done by the VS Code Dart extension, not `dart format`. I'll revert the newline change globally